setup.cfg

```diff
@@ -28,7 +28,7 @@ install_requires =
     click>=8.0,<9.0
     importlib-metadata>=4.0
     python-dotenv>=0.19.2
-    starknet-devnet>=0.3.5
+    starknet-devnet>=0.4.2
```

Comment:
The devnet is up to 0.4.3 from a couple weeks ago. Should we just bump it up now?